modules/i18n/src/models/namespaces/applications-ns.ts

```diff
@@ -273,6 +273,12 @@ export interface ApplicationsNS {
             content: string;
             assertionHint: string;
         };
+        InProgressReshare: {
```

Comment:
Let's be consistent with the other property names

```suggestion
        inProgressReshare: {
```